src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java

```diff
@@ -118,6 +110,11 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 
 		return true;
 	}
+
+	static boolean helpCommand(CommandSender sender, String[] args, List<String> adminHelp, List<String> playerHelp) {
```

Comment:
shouldn't this method be better suited to put in Common class (since its shared) or AdminCommand (Since its check for `.admin` perm node)

Reply:
Yeah probs, I'll adjust it when I have some time